modules/azure/aks/aks.tf

```diff
@@ -53,30 +53,22 @@ resource "azurerm_kubernetes_cluster" "this" {
   default_node_pool {
     name           = "default"
     vnet_subnet_id = data.azurerm_subnet.this.id
+    zones          = ["1", "2", "3"]
 
-    type                         = "VirtualMachineScaleSets"
-    zones                        = ["1", "2", "3"]
-    enable_auto_scaling          = false
-    only_critical_addons_enabled = true
-
-    orchestrator_version = var.aks_config.default_node_pool.orchestrator_version
-    node_count           = 1
+    orchestrator_version = var.aks_config.version
     # This is a bug in tflint
     # tflint-ignore: azurerm_kubernetes_cluster_default_node_pool_invalid_vm_size
-    vm_size = "Standard_D2as_v4"
-
-    node_labels = var.aks_config.default_node_pool.node_labels
+    vm_size                      = "Standard_D2ds_v5"
```

Comment:
Wrote the same thing in the other PR. 
What happens if this vm type runs out? I don't want to be forced to cut a new release to solve that at 3 in the night to get a cluster up and running.

I would like to see this as a variable with Standard_D2ds_v5 as the default value.

Reply:
Dont see how this an issue as as we are not provisioning default node pools late at night. We have so far never experienced an issue with the general VM instances and cannot see how we will in the future. I would rather not make this configurable but if it is an issue for you I am fine with changing it.

Reply:
That is very true at the same time we could definitely do a blue green at the middle of the night.
Yeah I think it would feel better for me if I knew I could overwrite this.
@jimgus we are talking about this in this PR instead.